storyscript/compiler/lowering/Lowering.py

```diff
@@ -584,6 +605,45 @@ def visit_function_dot(self, node, block):
         for c in node.children:
             self.visit_function_dot(c, block)
 
+    def visit_path(self, node, block):
+        """
+        Visit path's with expression and lower these expressions to path's.
+        """
+        if not hasattr(node, 'children') or len(node.children) == 0:
+            return
+
+        if node.data == 'block':
+            # only generate a fake_block once for every line
+            # node: block in which the fake assignments should be inserted
+            block = self.fake_tree(node)
+        elif node.data == 'path':
+            for child in node.children:
+                if not isinstance(child, Tree):
```

Comment:
```suggestion
                if not isinstance(child, Tree) or child.data != 'path_fragment':
```

then you get rid of one level of indentation.